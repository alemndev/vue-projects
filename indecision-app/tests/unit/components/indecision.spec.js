import { shallowMount, mount } from '@vue/test-utils'
import Indecision from '@/components/Indecision.vue'

describe('Indecision component', () => {
  // let wrapper
  // // Each test
  // beforeEach(() => {
  //   wrapper = shallowMount(Indecision)
  // })
  // test('check', () => {
  //   // const wrapper = shallowMount(Indecision)
  //   // expect(wrapper.html()).toMatchSnapshot()
  // })
  //test('render a h2 tag', () => {
  //  //const wrapper = shallowMount(Indecision)
  //
  //  // events
  //  //wrapper.trigger('click');
  //})
  // Read props
  //test('set default value', () => {
  //  //wrapper.props()
  // //wrapper.props('prop')
  //})
  // test('send props', () => {
  //   const wrapper = shallowMount(Indecision, { props: { my_prop: '.' } })
  // })
})
