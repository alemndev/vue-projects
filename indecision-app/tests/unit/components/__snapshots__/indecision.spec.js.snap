// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Indecision component check 1`] = `
<img alt="bg">
<div class="bg-dark"></div>
<div class="indecision-container"><input type="text" placeholder="A question">
  <p>Remember to end each question with a question mark (?)</p>
  <!--v-if-->
</div>
`;
